packages/superset-ui-connection/src/SupersetClientClass.ts

```diff
@@ -97,9 +105,9 @@ export default class SupersetClientClass {
     stringify,
     timeout,
     url,
-  }: RequestConfig): Promise<SupersetClientResponse> {
+  }: RequestConfig) {
     return this.ensureAuth().then(() =>
-      callApi({
+      callApiAndParseWithTimeout<T>({
```

Comment:
This seems to add timeout?

Reply:
It was the same function as before. `callApi/index.ts' actually exports `callApiAndParseWithTimeout` as default. I just made the import/export more transparent.